third_party/chromium/BUILD.bazel

```diff
@@ -4,4 +4,29 @@
 
 package(default_visibility = ["//visibility:public"])
 
-exports_files(glob(["**"]))
+load("@rules_foreign_cc//foreign_cc:make.bzl", "make")
+
+make(
+    name = "hyperdebug",
+    args = [
+        "-j",
+        "BOARD=hyperdebug",
+        "ALLOW_CONFIG=1",
+        "BUILDCC_PREFIX=x86_64-linux-gnu-",
+        "CROSS_COMPILE_arm=$$EXT_BUILD_ROOT$$/$(dirname $(execpath @arm_none_eabi//:gcc))/",
```

Comment:
The EC Makefile has slightly changed, such that the argument now is called "CROSS_COMPILE" (without the "_arm" suffix).  This change triggers a merge conflict in the other file where this PR moves the code from.

Reply:
Thanks, fixed